### ISSUE TITLE
Delay Random Spawns From the System You Just Arrived From

### ISSUE THREAD

**Issue by jafdy** [endless-sky/endless-sky#3432]
The idea is to prevent a newly spawned hostile ship from arriving just behind you (unless it was in the system you just came from, in witch case fair enough) by disallowing or heavily disfavoring your arrival path for a short time after you arrive. This *might* give new players a bit more breathing room and would add a tiny bit of realism. 

The obvious disadvantage is that this requires code time and may have no real effect at all, and there are systems with only one hyperspace lane and no landable planets.

**Comment by endless-sky** [comment]
I suspect that hostile ships that spawn entering behind you (not counting hostiles that were already in the previous system when you left it, and just jumped in your same direction) would account for at most 10-20% of new player deaths. Hostile ships spawning in the destination system are probably at least 50-70%. So, this would have only a tiny effect on how "safe" the game is.

**Comment by Hadron1776** [comment]
I don't think the idea is that new players don't die, but rather that they wouldn't die _before they can react._
(IMO, if ships instakill you the moment you jumped into a system, that's bad. If the player dies because they've lingered in the system for 5 minutes, that's their own fault for overstaying their welcome).

**Comment by endless-sky** [comment]
If you're dying before you can react, then the ship probably jumped less than 2 seconds behind you, meaning it's coming from the previous system (not randomly spawning).

A randomly spawning ship will start out in a different system with a random facing angle, so it will take a minimum of 4 seconds from the moment it is created to when it finishes jumping into the system. That means that at minimum, it will arrive two seconds after the player, and even that tight a timing will be very unlikely.

**Comment by KiLEdEnNis** [comment]
my issue is mostly jumping in as a starter ship and having a Pirate Leviathan (for example) laser me because I have to fly through it before I can really change course
that and insta Heavy Rockets, I hate Heavy Rockets
I hardly ever noticed this issue where a new enemy jumps in right after ya and I played quite a lot

**Comment by jafdy** [comment]
Yeah, I've never had it either but enough people complain about it that I think something should be done. Yeah, most ships that kill you come from the centre but it is just really annoying having a ship kill you from behind so confirmation bias kicks in and the perception is that the things that kill you are the ones that jump in from behind. 
I guess another side of the issue is if you just ran away from some pirates then they are no longer in combat and are free to promptly follow you from the system you just came from.
